core/src/miner/mod.rs

```diff
@@ -197,3 +198,14 @@ pub enum TransactionImportResult {
 
 #[cfg(all(feature = "nightly", test))]
 mod mem_pool_benches;
+
+fn fetch_account_creator<'c>(client: &'c dyn AccountData) -> impl Fn(&Public) -> AccountDetails + 'c {
```

Comment:
Why does this commit contain this change?

Reply:
@sgkim126 Sorry for my mistake. I made a mistake because I wasn't used to using git. I solved it with @majecty 's help. Please check and review again.